styleguide-app/Examples/Icon.elm

```diff
@@ -1,179 +1,201 @@
-module Examples.Icon exposing (example, styles)
+module Examples.Icon exposing (example)
 
 {-|
 
 @docs example, styles
 
 -}
 
-import Assets exposing (assets)
+import Assets exposing (Assets, assets)
 import Css
-import Css.Foreign exposing (Snippet, adjacentSiblings, children, class, descendants, each, everything, media, selector, withClass)
-import Headings
-import Html exposing (Html)
-import Html.Attributes exposing (style, title)
+import Css.Foreign
+import Html.Styled as Html exposing (Html)
+import Html.Styled.Attributes exposing (css, style, title)
 import ModuleExample exposing (Category(..), ModuleExample)
 import Nri.Ui.Colors.V1 exposing (..)
 import Nri.Ui.Icon.V2 as Icon
-import Nri.Ui.Styles.V1
 
 
 {-| -}
 example : ModuleExample msg
 example =
-    { filename = "Nri/Ui/Icon/V1.elm"
+    { filename = "ui/src/Nri/Icon.elm"
```

Comment:
I think it would actually be nice to have some version info in the style guide. I wonder if we should create a new styleguide page whenever we bump the version. I think that's not part of this pr, but maybe worth creating an issue/discourse.